docs/support/faq.md

```diff
@@ -136,97 +124,81 @@ Yes! See [boolean logic](../sdk/access-control/condition-types/boolean-logic) fo
-
-### 6. Orphaned and unreachable data due to nodes rotating
-
-**(cont'd) Even with the redundancy that 2/3 threshold encryption gives you, that's 2/3 of the number of shares at the time of the encryption right? So over a long enough time horizon, isn't it fairly likely that after a few years you'll have had enough nodes rotate off the network that more than a third of the shares for some early content encrypted by the network are now lost, leaving that data orphaned and unable to ever be decrypted?**
-
-Nope! you're probably thinking of shares in the context of shamir's secret sharing? We use threshold encryption which is different. Nodes all share one big private key generated via a distributed key generation operation. Nobody knows the whole private key. As nodes join and leave the network, through a process called proactive secret sharing, the private key shares are regenerated and each node gets a new private key share. But the shares, together, still represent the original private key.
-
-So the entire network could turnover (all nodes that were there when you encrypted content are gone) but you can still decrypt the content, becuause the private key itself is persisted
-
-### 7. So you need 2/3 of the entire network to decrypt content? not 2/3 of some fixed constant number of key fragments?
+In the federated network with named nodes, we know who the operators are, so a sybil attack is pretty hard. In the permissionless network, node operators must stake to run a node which means, you can do the math to figure out the cryptoeconomic guarantees depending on the number of nodes and the required stake amount, which are parameters that will be fine-tuned. We also intend to use probabilistic guarantees that make it difficult for a given node operator to actually know which private key shares they have and which public keys they correspond to. Meaning, a node operator doesn't actually know what stuff the key share they are holding will unlock. Additionally, the use of SEV-SNP as a hardware requirement means node operators ever have access to any of the underlying key material contained within their node.
 
-Yup! those are the default params we are launching with. Over time, we want to let users launch their own "subnets" with their own parameters.
+### 6. So you need 2/3 of the entire network to decrypt or sign the content? Not 2/3 of some fixed constant number of key fragments?
 
-### 8. As the number of nodes on the network grows, it gets more secure, but also slower to decrypt content?
+Yup! Those are the default params we are launching with. Over time, we want to let users launch their own "subnets" with their parameters.
```

Comment:
consider: 

'Yes, those are the default parameters that the Lit Network has launched with. We are currently exploring a 'subnet' architecture which would allow developers to launch their own subnets with their desired parameters'

Reply:
Done :) @a1ttech